core/src/main/java/com/amplifyframework/datastore/DataStoreChannelEventName.java

```diff
@@ -52,7 +52,22 @@
     /**
      * The remote synchonization processes stopped.
      */
-    REMOTE_SYNC_STOPPED("remote_sync_stopped");
+    REMOTE_SYNC_STOPPED("remote_sync_stopped"),
+
+    /**
+     * The DataStore is about to start the Sync Queries.
+     */
+    SYNC_QUERIES_STARTED("syncQueriesStarted"),
+
+    /**
+     * All models have been synced.
+     */
+    SYNC_QUERIES_READY("syncQueriesReady"),
+
+    /**
+     * All models have been synced.
+     */
+    MODEL_SYNCED("modelSynced");
```

Comment:
One says model (singular) and one says models (plural). From the Javadoc, it looks like it means the plural form. If that's so, can we name this `ALL_MODELS_SYCNED`? This probably means going back to the cross-platform design doc and making sure the other guys do likewise. Or, if JS already has these publicly documented and live, we could at least align some better names with iOS.

Reply:
This event is emitted once for each individual model, so it makes sense for it to be singular.

Regarding the event names in general, I'll check in with iOS to see how they're going about implementing. My understanding from one of the meeting we had is that they were going to follow JS though.